sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/IntegrationTestBase.java

```diff
@@ -204,46 +173,65 @@ public ProxyOptions getProxyConfiguration() {
     }
 
     /**
-     * Creates a new instance of {@link ServiceBusClientBuilder} with the default integration test settings and uses a
-     * connection string to authenticate.
+     * Creates a new instance of {@link ServiceBusClientBuilder} with authentication set up in {@link TestMode#LIVE} and
+     * {@link TestMode#RECORD} modes.
+     *
+     * @return the builder with authentication set up.
+     * @throws org.opentest4j.TestAbortedException if the test mode is {@link TestMode#PLAYBACK}.
      */
-    protected ServiceBusClientBuilder getBuilder() {
-        return getBuilder(false);
+    protected ServiceBusClientBuilder getAuthenticatedBuilder() {
+        final TestMode mode = super.getTestMode();
+        final ServiceBusClientBuilder builder = new ServiceBusClientBuilder();
+        if (mode == TestMode.LIVE) {
+            final String fullyQualifiedDomainName = TestUtils.getLiveFullyQualifiedDomainName();
+            assumeTrue(!CoreUtils.isNullOrEmpty(fullyQualifiedDomainName), "FullyQualifiedDomainName is not set.");
+            final TokenCredential credential = TestUtils.getPipelineCredential(credentialCached);
+            return builder.credential(fullyQualifiedDomainName, credential);
```

Comment:
Here following the recommended pattern of using pipeline credentials when running in LIVE mode,